hedera-node/src/main/java/com/hedera/services/store/models/Topic.java

```diff
@@ -86,6 +91,46 @@ public static Topic fromGrpcTopicCreate(
 		return topic;
 	}
 
+	/**
+	 * <p>Updates the topic model</p>
+	 * <p><b>Affected properties can be:</b></p>
+	 * <li>Admin key</li>
+	 * <li>Submit key</li>
+	 * <li>Memo</li>
+	 * <li>expirationTimestamp</li>
+	 * <li>autoRenewDurationSeconds</li>
+	 * <li>autoRenewAccountId</li>
+	 * @param op
+	 * 		<p>the Transaction body</p>
+	 * @param designatesAccountRemoval
+	<p>if the auto-renew account is designated for removal, sets the autoRenewAccountId to null.</p>
+	<p>Otherwise - replaces it with the autoRenewAccountId, contained in the transaction body.</p>
+	 */
+	//TODO: Write test for the bellow method and validate, that properly maps the Merkle topic properties to the model
```

Comment:
Is this TODO a leftover? If it is let's remove it

Reply:
No, this is the test, for which I mentioned in the today's Daily, that I forgot to write,